### ISSUE TITLE
[warning] failed package validation and/or copy for commit e64a5b87da963d4bce9488eb618d34efaf151f94

### ISSUE THREAD

**Issue by conda-forge-linter** [conda-forge/cinnabar-feedstock#1]
Hi @conda-forge/cinnabar! This is the friendly automated conda-forge-webservice!

It appears that one or more of your feedstock's outputs did not copy from the
staging channel (cf-staging) to the production channel (conda-forge). :(

This failure can happen for a lot of reasons, including an outdated feedstock
token. Below we have put some information about the failure to help you debug it.

**Rerendering the feedstock will usually fix these problems.**

If you have any issues or questions, you can find us on gitter in the
community [chat room](https://gitter.im/conda-forge/conda-forge.github.io) or you can bump us right here.

output validation (is this output allowed for your feedstock?):
 - **noarch/cinnabar-0.2.4-pyhd8ed1ab_0.conda**: False

copied (did this output get copied to the production channel?):
 - **noarch/cinnabar-0.2.4-pyhd8ed1ab_0.conda**: False

error messages:
 - output noarch/cinnabar-0.2.4-pyhd8ed1ab_0.conda not allowed for conda-forge/cinnabar-feedstock

To fix invalid outputs, you may need to manually add this feedstock to the feedstock-outputs map (did another feedstock already make this package?):
 - https://github.com/conda-forge/feedstock-outputs

**Comment by jakirkham** [comment]
This [looks odd]( https://github.com/conda-forge/feedstock-outputs/blob/ad643d52c19234e57ad21213998d55eabcb6606a/outputs/c/i/n/cinnabar.json#L1 ). Also [the `arsenic` feedstock packages `arsenic`]( https://github.com/conda-forge/arsenic-feedstock/blob/0b02c0918b7f4661e65008941ae6eca842d609ca/recipe/meta.yaml#L1 ) (not `cinnabar`).

@conda-forge/core, should we fix feedstock outputs here?

**Comment by beckermr** [comment]
Let's not change anything until we understand what happened.

**Comment by beckermr** [comment]
It looks like in resolving this issue https://github.com/conda-forge/arsenic-feedstock/issues/6 the maintainers switched the recipe to cinnabar in https://github.com/conda-forge/arsenic-feedstock/pull/8. They then switched the recipe back in https://github.com/conda-forge/arsenic-feedstock/pull/15. There is complete overlap in the maintainers except for one person. 

@conda-forge/arsenic @conda-forge/cinnabar Can you all explain what happened here?

FYI, we require packages new to conda-forge to have their own feedstock that goes through staged-recipes. It looks like at least at first that did not happen here. Not a big deal but please remember for the future.

**Comment by beckermr** [comment]
ahhhhh - https://github.com/conda-forge/staged-recipes/pull/21446#issuecomment-1338979487

We need to lint for outputs that overlap in current ones.

**Comment by beckermr** [comment]
Looks like we should add cinnabar to the allowed uploaders, but let's wait for confirmation.

**Comment by beckermr** [comment]
I opened this one: https://github.com/conda-forge/conda-smithy/issues/1705

**Comment by dwhswenson** [comment]
I think @mikemhenry is the main contact for this, but situation was pretty much the following:

1. Part of our org's mission is to develop/maintain open source software that academic developers don't have time to maintain. `arsenic` was one of those.
2. The name `arsenic` is taken on PyPI, so we renamed our fork to `cinnabar` (we're expecting major changes anyway). We probably should have made a new feedstock via staged-recipes at that time -- sorry!
3. It turns out that some old users don't like the API-breaking improvements we're making, and want the old `arsenic`. And hypothetically want to maintain it. So we that's when we got around to making the new feedstock, and switching the product of the old feedstock back to `arsenic`.

Going forward, `arsenic` is legacy and `cinnabar` is the future. We'll stay on as conda-forge maintainers of `arsenic` to point people over here, and possibly to help those who want to maintain the old fork get releases out.
